utils/utils.go

```diff
@@ -25,3 +25,22 @@ func ScaleFixed(number *big.Int) float64 {
 	retF, _ := new(big.Float).Quo(new(big.Float).SetInt(number), big.NewFloat(1e18)).Float64()
 	return retF
 }
+
+func BoolToMetric(value bool) float64 {
```

Comment:
Optional Nit: `BoolToMetric` name feels a bit off to me, but I honestly don't have a better suggestion